### DISCUSSION TITLE
Count loggedin users

### DISCUSSION THREAD

**Discussion by MrVibe** [supabase/supabase#29026] [Questions] [answered]
how can we count loggedin users in a site ? without a table with realtime connection, I suppose this information is already present with the gotrue client already.

**Comment by GaryAustin1** [comment, marked as answer]
What is a logged in user?
With Supabase auth you signin and then are basically signed in until you signout.  That could be months.

You might be able to use auth.sessions table to see what sessions are being refreshed in a time period to see how many user clients are open.  There is also the auth.refresh_token table that has an entry each time the user client asks to refresh.  You would have to filter out duplicates base on user id just based on a quick look.

**Reply by MrVibe** [reply]
My assumption is that anone whose token is refreshed 30minutes is a logged in user.

**Reply by GaryAustin1** [reply]
I would say past 1.x hours if you have jwt expire set to default of an hour.
